### ISSUE TITLE
Look into Prettier formatting for code samples

### ISSUE THREAD

**Issue by alexandratran** [MetaMask/metamask-docs#1122]
Look into using [Prettier](https://prettier.io/) for formatting the code samples in all Markdown files. May have to use a specific configuration to not lose certain Markdown formats we use (e.g., line breaks at a certain column limit). See the code guidelines used in https://github.com/MetaMask/metamask-docs/pull/1116.

Edit: We want to revert back to 2 space indents instead of 4 across the docs.

Edit 2: Installing [`eslint-plugin-markdown`](https://github.com/eslint/eslint-plugin-markdown) is blocked by requiring Node v18+. We need to update Docusaurus first (#1048).

**Comment by subhamkumarr** [comment]
@alexandratran For `sanp` and `wallet` code formatting is already done. `Service` section and few other markdown file shall I do the code formatting?
